src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -441,6 +445,9 @@ def _acquireTiles(self):
         da_list = []  # for each position, a list of DataArrays
         prev_idx = [0, 0]
         i = 0
+        # Make sure to begin from starting position
+        self._stage.moveAbs(self._starting_pos)
```

Comment:
This should wait for the move to complete.
Ideally, it'd be the same code as in _moveToTile(), but otherwise you could just use moveAbsSync()